src/coreComponents/functions/TableFunction.hpp

```diff
@@ -240,6 +250,14 @@ class TableFunction : public FunctionBase
    */
   virtual real64 evaluate( real64 const * const input ) const override final;
 
+  /**
+   * @brief Method to get coordinates
+   * @param input a scalar input
+   * @param interpolationMethod the interpolation method
+   * @return the coordinates
+   */
+  std::vector< real64 > getCoord( real64 const * const input, InterpolationType interpolationMethod ) const;
```

Comment:
```suggestion
  real64 getCoord( real64 const * const input, InterpolationType interpolationMethod, integer const dim = 0 ) const;
```
I would add the dimension to the input and simply return 1 value. It looks like that's what the application needs anyway.

Reply:
will check that

Reply:
done